onnxruntime/test/util/test_utils.cc

```diff
@@ -183,7 +183,7 @@ void SparseIndicesChecker(const ONNX_NAMESPACE::TensorProto& indices_proto, gsl:
         ASSERT_STATUS_OK(utils::UnpackInitializerData(indices_proto, model_path, unpack_buffer));
         ind_span = ReinterpretAsSpan<const int64_t>(gsl::make_span(unpack_buffer));
       } else {
-        ind_span = gsl::make_span(indices_proto.int64_data().cbegin(), indices_proto.int64_data().cend());
+        ind_span = gsl::make_span(indices_proto.int64_data().data(), (std::size_t)indices_proto.int64_data_size());
```

Comment:
The returned value type of int64_data_size()  is int? It should be smaller than size_t. Why this casting is needed?

Reply:
> The returned value type of int64_data_size() is int? It should be smaller than size_t. Why this casting is needed?
Yes, int64_data_size() is int. It is not needed, it is my mistake with template function match. I will remove this casting.